sound/soc/sof/intel/hda-loader.c

```diff
@@ -314,44 +314,49 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 	init_waitqueue_head(&sdev->boot_wait);
 	sdev->boot_complete = false;
 
-	/* try attempting fw boot a few times before giving up */
+	/* try attempting ROM init a few times before giving up */
```

Comment:
while at it you could also improve the wording a bit, I think "try attempting" is a bit of an overkill ;-)

Reply:
let me change it to 'try' as I am at it. :)